import-restrictions.yaml

```diff
@@ -4,13 +4,22 @@
     - github.com/docker/compose-cli/cli
     - github.com/docker/compose-cli/ecs
     - github.com/docker/compose-cli/local
+    - github.com/docker/compose-cli/v2/cli
+    - github.com/docker/compose-cli/v2/ecs
+    - github.com/docker/compose-cli/v2/local
```

Comment:
FWIW; looks like this configuration only looks at _literal_ paths, not subdirectories? (e.g. `aci/e2e` imports the CLI); https://github.com/docker/compose-cli/blob/7f18b47a9acfece3411bfada46d9565a7035d2bd/aci/e2e/e2e-aci_test.go#L53

And I think I saw some other paths as well